modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc

```diff
@@ -43,7 +43,7 @@ data:
       mirror-by-digest-only = true
 ----
 <1> The mirror registry’s certificate used when creating the mirror registry.
-<2> The configuration for the mirror registry.
+<2> The configuration file for the mirror registry. The mirror registry configuration adds mirror information to `/etc/containers/registries.conf` in the Discovery image. The mirror information is stored in the `imageContentSources` section of the `install-config.yaml` file when passed to the installer. The Assisted Service pod running on the HUB cluster configures the mirror registry to allow access. It then fetches images from the mirror registry. 
```

Comment:
Maybe change last 2 sentences to:
The Assisted Service pod running on the HUB cluster uses the configured mirror registry to fetch the container images.